mirascope/core/openai/call_params.py

```diff
@@ -11,17 +13,36 @@
 
 from ..base import BaseCallParams
 
+if TYPE_CHECKING:
+    from openai.types.chat.chat_completion_audio_param import (  # pyright: ignore [reportMissingImports]
```

Comment:
again, won't this fail if it's not the right version?

Reply:
Same above